tests/main_test.py

```diff
@@ -3,6 +3,7 @@
 from telegram import Message, Chat
 from datetime import datetime
 import src.main as main
+from random import randrange
```

Comment:
As we're already importing `src.main`, it should be better use main.randrange.

But, should we really use randrange?